### PR TITLE
chart/template: Fix tracing endpoint variable

### PR DESCRIPTION
**Pull request by ArthurSens**
## Description

The issue is never where we think it is 😅.

Turns out https://github.com/gitpod-io/ops/pull/347 didn't work because of a typo at our chart. I've already checked and we don't use `tracing.endoint` anywhere else, so it should be safe to merge.

Pinging @MrSimonEmms and @csweichel because on today's Tech talk you've mentioned that changes to our chart probably require changes to the installer as well

## Related Issue(s)

Fixes https://github.com/gitpod-io/ops/issues/340

## Release Notes

```release-note
Fix typo at Gitpod's helm chart
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

Associated issue: *#340*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mads-hartmann — APPROVED** [review]
LGTM :D

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: c9d6613c561a6a476f9c593e8109eb075fceca4e</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6933#pullrequestreview-817631275" title="Approved">mads-hartmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

Associated issue: *#340*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
New changes are detected. LGTM label has been removed.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 57224aff327aac3ad9526ff58c1784d2b5d6b5e5</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6933#issuecomment-981623013" title="LGTM">geropl</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6933#pullrequestreview-817631275" title="Approved">mads-hartmann</a>*

Associated issue: *#340*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
